test/Solnet.Rpc.Test/Resources/LogsSubscribeNotification.json

```diff
@@ -0,0 +1 @@
+{"jsonrpc":"2.0","method":"logsNotification","params":{"result":{"context":{"slot":5208469},"value":{"signature":"5h6xBEauJ3PK6SWCZ1PGjBvj8vDdWG3KpwATGy1ARAXFSDwt8GFXM7W5Ncn16wmqokgpiKRLuS83KUxyZyv2sUYv","err":null,"logs":["BPF program 83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri success"]}},"subscription":23784}}
```

Comment:
Where exactly are you subscribing to this notification? Because you either subscribe to `all` or subscribe to ```{"mentions":["83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri"]}```